sample-input/benchmarks/c5g7/c5g7-3d-rodded-B.py

```diff
@@ -0,0 +1,526 @@
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+            [1, 1, 1, 1, 1, 2, 1, 1, 2, 1, 1, 2, 1, 1, 1, 1, 1],
+            [1, 1, 1, 2, 1, 1, 1, 1, 1, 1, 1, 1, 1, 2, 1, 1, 1],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+            [1, 1, 2, 1, 1, 2, 1, 1, 2, 1, 1, 2, 1, 1, 2, 1, 1],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+            [1, 1, 2, 1, 1, 2, 1, 1, 3, 1, 1, 2, 1, 1, 2, 1, 1],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+            [1, 1, 2, 1, 1, 2, 1, 1, 2, 1, 1, 2, 1, 1, 2, 1, 1],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+            [1, 1, 1, 2, 1, 1, 1, 1, 1, 1, 1, 1, 1, 2, 1, 1, 1],
+            [1, 1, 1, 1, 1, 2, 1, 1, 2, 1, 1, 2, 1, 1, 1, 1, 1],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]]
+
+
+universes = {1 : reflector, 2 : control_rod, 3 : fission_chamber}
+for i in range(17):
+  for j in range(17):
+    template[i][j] = universes[template[i][j]]
+
+lattices[-1].setUniverses(template)
+assembly_rfl_rod_cell.setFill(lattices[-1])
+
+# Reflector unrodded 17 x 17 assemblies
+lattices.append(openmoc.Lattice(name='Assembly Reflector Unrodded'))
+lattices[-1].setWidth(width_x=1.26, width_y=1.26)
+template = [[refined_reflector] * 17] * 17
+lattices[-1].setUniverses(template)
+assembly_rfl_unrod_cell.setFill(lattices[-1])
+
+# Reflector unrodded assembly right
+lattices.append(openmoc.Lattice(name='Assembly Reflector Unrodded Right'))
+lattices[-1].setWidth(width_x=1.26, width_y=1.26)
+template = [[refined_reflector] * 11 + [reflector] * 6] * 17
+lattices[-1].setUniverses(template)
+assembly_rfl_unrod_cell_rgt.setFill(lattices[-1])
+
+# Reflector unrodded assembly bottom
+lattices.append(openmoc.Lattice(name='Assembly Reflector Unrodded Bottom'))
+lattices[-1].setWidth(width_x=1.26, width_y=1.26)
+template = [[refined_reflector] * 17] * 11
+template += [[reflector] * 17] * 6
+lattices[-1].setUniverses(template)
+assembly_rfl_unrod_cell_btm.setFill(lattices[-1])
+
+# Reflector unrodded assembly bottom
+lattices.append(openmoc.Lattice(name='Assembly Reflector Unrodded Corner'))
+lattices[-1].setWidth(width_x=1.26, width_y=1.26)
+template = [[refined_reflector] * 11 + [reflector] * 6] * 11
+template += [[reflector] * 17] * 6
+lattices[-1].setUniverses(template)
+assembly_rfl_unrod_cell_cnr.setFill(lattices[-1])
+
+
+###############################################################################
+#########################   Creating Core Lattice   ###########################
+###############################################################################
+
+# 3 x 3 x 9 core to represent 3D core
+lattices.append(openmoc.Lattice(name='Full Geometry'))
```

Comment:
Can this be shortened as:

```python
lattices[-1].setUniverses
([[[assembly_rfl_rod      , assembly_rfl_rod      , assembly_rfl_unrod_rgt],
   [assembly_rfl_rod      , assembly_rfl_rod      , assembly_rfl_unrod_rgt],
   [assembly_rfl_unrod_btm, assembly_rfl_unrod_btm, assembly_rfl_unrod_cnr]]*3,
  [[assembly_uo2_rod      , assembly_mox_rod      , assembly_rfl_unrod_rgt],
   [assembly_mox_rod      , assembly_uo2_unrod    , assembly_rfl_unrod_rgt],
   [assembly_rfl_unrod_btm, assembly_rfl_unrod_btm, assembly_rfl_unrod_cnr]]*2,
  [[assembly_uo2_unrod    , assembly_mox_unrod    , assembly_rfl_unrod_rgt],
   [assembly_mox_unrod    , assembly_uo2_unrod    , assembly_rfl_unrod_rgt],
   [assembly_rfl_unrod_btm, assembly_rfl_unrod_btm, assembly_rfl_unrod_cnr]]*4])
```

Again, this would assume we stick with two different ``Lattice::setUniverses(...)`` routines.